cli/.prettierrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "semi": true,
+  "semi": false,
```

Comment:
Why was this changed?
It'll likely case a lot of formatting changes if we have TS/JS files

Reply:
This should have been the default, this is what we have in the root config already and in other repos. I'll run formatting to ensure it's consistent.

Reply:
There's other differences with this one (which is used for the CLI project), we could discuss using the same one but I'd do it separately personally

Reply:
Yes, will do it separately to ensure consistency across the mono repo.